examples/CMakeLists.txt

```diff
@@ -6,3 +6,4 @@ add_subdirectory(simple)
 add_subdirectory(batch)
 add_subdirectory(metrics_simple)
 add_subdirectory(multithreaded)
+add_subdirectory(w3c_tracecontext_test)
```

Comment:
Put w3c_tracecontext_test into some test folder instead of under examples?

Reply:
Yes, I'll move it. Probably I'll set up something similar to what C# people did: https://github.com/open-telemetry/opentelemetry-dotnet/tree/master/test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests

Reply:
Yes, docker images or script to run both the services and give result would be handy ( as part of separate PR )

Reply:
Right, would be good to have a script / docker image to run both the services and give the result.

Reply:
Yes, docker images or script to run both the services and give result would be handy ( as part of separate PR )

Reply:
This is moved now into `ext`, signaling that this is not a central component or test that has to run with the core test suite.